.github/workflows/update_github_io.yml

```diff
@@ -0,0 +1,31 @@
+#############################################################################
+# GitHub Action to Update github.io
+#
+#############################################################################
+name: "R: update github.io"
```

Comment:
`Update`

In all other actions we use capital, let's be consistent

Reply:
Changes Done